app/views/static/contact.html.erb

```diff
@@ -1,11 +1,63 @@
-<% breadcrumb_title = "Contact Us"; breadcrumb_link = '/contact' %>
-<% breadcrumb_title_abbr = nil %>
-<%= render "static/content_chunk",
-      curr_page: true,
-      breadcrumb_title: breadcrumb_title,
-      breadcrumb_title_abbr: breadcrumb_title_abbr,
-      breadcrumb_link: breadcrumb_link, 
-      title:
-        link_to("<h2 class='static-h2'>#{breadcrumb_title}</h2>".html_safe, breadcrumb_link),
-      bod_text: "Placeholder for copy"
-%>
\ No newline at end of file
+<% title = I18n.t('contact'); abbr = title.split(' ').first(3).join(' ') + '...' %>
```

Comment:
Since the title is only three words long, it doesn't need to be abbreviated, and the `abbr` local can be set to `nil`.